script/bootstrap

```diff
@@ -54,6 +54,12 @@ install_packages_apt()
 
     # mDNS
     sudo apt-get install --no-install-recommends -y libavahi-client3 libavahi-common-dev libavahi-client-dev avahi-daemon
+    (MDNS_RESPONDER_SOURCE_NAME=mDNSResponder-878.30.4 \
+        && cd /tmp \
+        && wget -4 --no-check-certificate https://opensource.apple.com/tarballs/mDNSResponder/$MDNS_RESPONDER_SOURCE_NAME.tar.gz \
+        && tar xvf $MDNS_RESPONDER_SOURCE_NAME.tar.gz \
+        && cd $MDNS_RESPONDER_SOURCE_NAME/mDNSPosix \
+        && make os=linux && sudo make install os=linux)
```

Comment:
If avahi is not used by our reference release, we should remove these avahi packages to save image space.

Also, should we install one of mDSNResponder and Avahi instead of installing both?

Reply:
The `bootstrap` script is not only for creating images. All OTBR developer's will need to run the script to set up their environment, such as who working on `avahi` mDNS implementations.

Reply:
It's apparently not ideal to install two mDNS apps, especially for the reference BR release, which will always use mDNSResponder and should better uninstall Avahi packages. 

I wonder if `installing mDNS` should be part of `setup` script rather than `bootstrap`.

Thoughts? @wgtdkp @bukepo

Reply:
There will be a dozen dependencies that are not necessary if some options are not enabled, and it will be too much to define an variable for each such case...  I don't feel this really necessary.

Reply:
Anyway, we will switch to `avahi` when tests for avahi version is added and we can have the mDNSResponder library removed from here.